routers/repo/view.go

```diff
@@ -210,9 +211,26 @@ func renderFile(ctx *context.Context, entry *git.TreeEntry, treeLink, rawLink st
 			}
 
 			var output bytes.Buffer
-			lines := strings.Split(fileContent, "\n")
+			var terminator string
+
+			if strings.Index(fileContent, "\r\n") != -1 {
+				lf, _ := regexp.MatchString("[^\r]\n", fileContent)
+				if lf {
+					terminator = "\n"
+				} else {
+					terminator = "\r\n"
+				}
+			} else {
+				terminator = "\n"
+			}
+			lines := strings.Split(fileContent, terminator)
```

Comment:
Yeah, but how do I with which character to join the lines afterwards in the loop? I'd like to preserve CRLF if it's there.